capitains_nautilus/cts/collections.py

```diff
@@ -205,12 +199,14 @@ def decide_class(self, key):
 
 
 class SparqlXmlCtsTextgroupMetadata(CTSSparqlNavigatedCollection, XmlCtsTextgroupMetadata):
```

Comment:
`CTSSparqlNavigatedCollection` should probably be `CtsSparqlNavigatedCollection`